composer.json

```diff
@@ -13,11 +13,12 @@
         "league/plates": "^3.3",
         "php-di/php-di": "^6.0",
         "nikic/fast-route": "^1.3",
-        "nyholm/psr7": "^1.3",
-        "nyholm/psr7-server": "^1.0"
+        "nyholm/psr7": "^1.4",
+        "nyholm/psr7-server": "^1.0",
+        "psr/log": "^1.1"
     },
```

Comment:
I'm actually using composer `2.1.5` and I don't have any notice. Are you still using composer v1? 
We don't need to use `Prestissimo` because it works only for composer v1.

Reply:
I'm not using running locally. Check the "Checks" tab in Github for the 4 checks. Likely a mute point.